hackathon-for-hunger/Modules/User/Views/ProfileViewController.swift

```diff
@@ -10,10 +10,50 @@ import UIKit
 
 class ProfileViewController: UIViewController {
     
+
+    private let userService: UserService = UserService()
+
+ 
+    @IBOutlet weak var userNameTF: UITextField!
+    @IBOutlet weak var emailTF: UITextField!
+    
+    @IBAction func save(sender: AnyObject) {
+        print("Save pressed")
+        updateUser()
+    }
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         self.setupMenuBar()
         self.title = "Profile"
+        let currentUser = AuthService.sharedInstance.getCurrentUser()
+        userNameTF.text = currentUser?.name
+        emailTF.text = currentUser?.email
+
+
+    }
+    
+    func updateUser() {
+        
+        
+        var user = UserRegistration()
```

Comment:
the user registration model is for registration only. you need to make a user update model to represent the transaction. it may seem like duplicaiton, but it is domain driven descriptive coding with intent